src/dotnet/ToolManifest/LocalizableStrings.resx

```diff
@@ -118,32 +118,33 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="CannotFindAnyManifestsFileSearched" xml:space="preserve">
-    <value>Cannot find any manifests file. Searched {0}</value>
+    <value>Cannot find any manifests file. Searched
+{0}</value>
   </data>
   <data name="FieldCommandsIsMissing" xml:space="preserve">
-    <value>Field commands is missing.</value>
+    <value>Missing 'commands' entry.</value>
   </data>
   <data name="InvalidManifestFilePrefix" xml:space="preserve">
-    <value>Invalid manifest file.
-{0}</value>
+    <value>Invalid manifest file. Path: {0}
```

Comment:
Seems to have a line break here as well.

Reply:
yes, it is intentional